[16.0][ADD] stock_release_channel_cutoff (#740)
OCA/wms: stock_release_channel_cutoff/readme/DESCRIPTION.rst

> This module lets users select a release channel and set a specific cutoff time for it.

Reviewer: ```suggestion
This module allows to set a cutoff time on a release channel.

```

> When the current time (now) becoms later than the cutoff time, the kanban view for that channel will display the cutoff time in red as a warning.

Reviewer: ```suggestion
When the current time (now) becomes later than the cutoff time, and the channel is still open, the kanban view for that channel will display the cutoff time in red as a warning.
```
